config/config.json

```diff
@@ -4,6 +4,9 @@
   "prefix": "-",
   "rotatePlayingGame": true,
   "rotatePlayingGameTime" : 150000,
+  "rotateStreamingStatus": false,
+  "rotateStreamingStatusTime" : 150000,
+  "Streaming/Game NOTE:" : "DO NOT SET BOTH STREAM AND GAME ROTATION TO TRUE AT THE SAME TIME",
```

Comment:
No need for the `Streaming/Game NOTE:`, we can add a check to see if the user is using both at the same time. This line can be deleted.